### PR TITLE
[8.x] Replace bootstrap file with phpunit INI values

### PR DESCRIPTION
**Pull request by lucasmichot**
Replace bootstrap file with phpunit INI values.

### CODE REVIEW

File: tests/bootstrap.php [hunk 1]
```diff
@@ -1,34 +0,0 @@
...
-|
-| Here we will set the default timezone for PHP. PHP is notoriously mean
-| if the timezone is not explicitly set. This will be used by each of
-| the PHP date and date-time functions throughout the application.
-|
-*/
-
-date_default_timezone_set('UTC');
-
-Carbon::setTestNow(Carbon::now());
```
Comment (driesvints) [inline tests/bootstrap.php:32]: What happens with this?

Comment (driesvints) [reply tests/bootstrap.php:32]: Tests seem to pass so this lgtm.

Comment (lucasmichot) [reply tests/bootstrap.php:32]: > What happens with this?
This is useless

**Comment by driesvints** [comment]
@lucasmichot any reason why this is still in draft? Please note that draft PR's aren't reviewed: https://laravel.com/docs/8.x/contributions#bug-reports

**Review by driesvints — APPROVED** [review]

**Comment by lucasmichot** [comment]
> @lucasmichot any reason why this is still in draft? Please note that draft PR's aren't reviewed: https://laravel.com/docs/8.x/contributions#bug-reports

Ah my bad @driesvints - no reason at all, I have just been forgetful with this one

File: phpunit.xml.dist [hunk 2]
```diff
@@ -2,7 +2,6 @@
 <phpunit backupGlobals="false"
          backupStaticAttributes="false"
          beStrictAboutTestsThatDoNotTestAnything="false"
-         bootstrap="tests/bootstrap.php"
```
Comment (driesvints) [inline phpunit.xml.dist:5]: Btw, shouldn't this be:

```
 bootstrap="vendor/autoload.php"
```

Comment (driesvints) [reply phpunit.xml.dist:5]: I guess not because all tests pass but I'm a bit surprised this works out of the box 😄

Comment (lucasmichot) [reply phpunit.xml.dist:5]: `bootstrap="vendor/autoload.php"` can be omitted as, this is the default value.

Comment (driesvints) [reply phpunit.xml.dist:5]: TIL :)

**Comment by driesvints** [comment]
@lucasmichot thanks man. This was a good PR 👍

**Pull request merged**